conf/services.config

```diff
@@ -16,16 +16,17 @@
     system (eg. sendmail or mail)."
 
     In the environments spun by Nextflow Tower on AWS Batch, the underlying
-    sendmail or mail calls are not available, so you have to specify SMPT
+    sendmail or mail calls are not available, so you have to specify SMTP
```

Comment:
were these just typos, or is SMPT different from SMTP?

Reply:
typos